src/components/GraphEditor/Editors/MoreInfosEditor/CheckAdjacency/CheckAdjacency.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useCy } from '../../../../../providers/useCy';
+import InstructionBox from '../../../EditorContainer/components/InstructionBox/InstructionBox';
+import { Container, InfoBox, InfoLabel, StyledButton } from './styles';
+
+const CheckAdjacency: React.FC = () => {
+    const cy = useCy();
+
+    const [sourceNode, setSourceNode] = useState<string>('');
+    const [targetNode, setTargetNode] = useState<string>('');
+    const [isAdjacent, setIsAdjacent] = useState<boolean>(false);
+
+
+    const registerNode = useCallback((node: any) => {
+      if (sourceNode === '') {
+        setSourceNode(node);
+        return;
+      }
+      setTargetNode(node);
+    }, [sourceNode, setTargetNode]);
+  
+    const handleTap = (e: any) => registerNode(e.target.id());
+  
+    useEffect(() => {
+      const cyRef = cy.current;
+  
+      cyRef.nodes().on('tap', handleTap);
+
+      checkAdjacency();
+  
+      return () => {
+        cyRef.nodes().off('tap', handleTap);
+      };
+    }, [cy, handleTap]);
+
+    const checkAdjacency = (() => {
+        const cyRef = cy.current;
+
+        const source = cyRef.getElementById(sourceNode)
+        const target = cyRef.getElementById(targetNode)
+        const isAdj = source.neighborhood().contains(target)
+
+        setIsAdjacent(isAdj);
+    });
+
+    const clearSelection = (() => {
+      setSourceNode('');
+      setTargetNode('');
+    });
+
+    const getNodeLabel = ((nodeId: string) => {
+      const cyRef = cy.current;
+      const nodeLabel = cyRef.getElementById(nodeId).data('label');
+
+      return nodeLabel;
+    });
+
+    return (
+      <>
+        <Container>
+          {sourceNode === '' && <InstructionBox content={'Click source node'} />}
+          {sourceNode !== '' && targetNode === '' && <InstructionBox content={'Click target node'} />}

```

Comment:
61 and 62 should be above <Container>

Currently margin-top: 30px is being applied to the InstructionBoxes, causing them to go lower than expected